tracee/event_monitor_ebpf.c

```diff
@@ -742,22 +741,29 @@ static __always_inline int save_context_to_buf(submit_buf_t *submit_p, void *ptr
 
 static __always_inline int save_to_submit_buf(submit_buf_t *submit_p, void *ptr, int size, u8 type)
 {
-    if (submit_p->off > SUBMIT_BUFSIZE_HALF)
-        // not enough space - return
-        return 0;
+// The biggest element that can be saved with this function should be defined here
+#define MAX_ELEMENT_SIZE sizeof(struct sockaddr_un)
```

Comment:
why do we use this and not the `size` arg?

Reply:
Because the BPF validator needs to know the maximal size in COMPILE time and not in run time, otherwise, it will refuse to run the program